boa_engine/src/bytecompiler/jump_control.rs

```diff
@@ -9,38 +9,157 @@
 //! [try spec]: https://tc39.es/ecma262/#sec-try-statement
 //! [labelled spec]: https://tc39.es/ecma262/#sec-labelled-statements
 
-use crate::bytecompiler::{ByteCompiler, Label};
+use crate::{
+    bytecompiler::{ByteCompiler, Label},
+    vm::{Handler, Opcode},
+};
 use bitflags::bitflags;
 use boa_interner::Sym;
 
+/// An actions to be performed for the local control flow.
+#[derive(Debug, Clone, Copy)]
+pub(crate) enum JumpRecordAction {
+    /// Places a [`Opcode::Jump`], transfters to a specified [`JumpControlInfo`] to be handled when it gets poped.
+    Transfter {
```

Comment:
Just want to confirm, is this meant to be `Transfter` vs. `Transfer`.

Reply:
Typo, definitely `Transfer`.